bin/import-sql

```diff
@@ -47,7 +47,7 @@ function import_all_sql_files() {
       # Run import-sql script in parallel, up to MAX_PARALLEL_PSQL processes at the same time
       : "${MAX_PARALLEL_PSQL:=5}"
       echo "Importing $(find "$dir/parallel" -name "*.sql" | wc -l) sql files from $dir/parallel/, up to $MAX_PARALLEL_PSQL files at the same time"
-      find "$dir/parallel" -name "*.sql" -print0 | xargs -0 -I{} -P "$MAX_PARALLEL_PSQL" "$0" "{}"
+      find "$dir/parallel" -name "*.sql" -print0 | xargs -0 -I{} -P "$MAX_PARALLEL_PSQL" sh -c "\"$0\" \"{}\" || exit 255"
```

Comment:
is it possible to use single quotes here?  If so, would make it cleaner

Reply:
`sh -c '\"$0\" \"{}\" || exit 255'` rise me an error `"sh": not found`

Reply:
you wouldn't need to escape the double quotes - `sh -c '"$0" "{}" || exit 255'`

Reply:
No, in this way $0 inside single quote it not interpreted.

Reply:
good point thx.

Reply:
> `sh -c '\"$0\" \"{}\" || exit 255'` rise me an error `"sh": not found`

No idea why `sh` is missing on your side. I can replace `sh` by `/usr/bin/env bash`?

Reply:
It was when I test switch to single quotes. No problem as it is now. Thanks